### ISSUE TITLE
Doubles girl in Idols

### ISSUE THREAD

**Issue by AkariHimine** [MagiCircles/CinderellaProducers#39]
There is Chitose Kurosaki (https://cinderella.pro/idol/327/Chitose-Kurosaki/) and Kurosaki Chitose (https://cinderella.pro/idol/335/Kurosaki-Chitose/)

**Comment by AyaxAki** [comment]
Yeah i saw it yesterday too...they are double. The doubles are still there.
